### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by ElchinNasirov**
## Description

In this Pull Request I worked with Alex to create the landing page of the project. Worked on RenderLandingPage.js to create the similar landing view page to the sample one from stakeholders. 
Added header, some text, 3 different buttons tried to mimic the sample one. Also, tried to match the colors and styles as well. Only the background picture is missing which will be added shortly.

## Video 
https://www.loom.com/share/5aed441a21ec44dd95d9c1217c5cd675?sharedAppSource=personal_library

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Landing/RenderLandingPage.js [hunk 1]
```diff
@@ -1,19 +1,75 @@
...
-        </p>
+        </p> */}
+
+      {/* Main text container */}
+      <div className="landing-main-container">
+        <div className="landing-main-text">
+          <h1>Coding is a superpower.</h1>
+          <h3>
+            {' '}
+            Through a love of teachin g and technology, we empower youth to
```
Comment (Ryan-E-Mark) [inline src/components/pages/Landing/RenderLandingPage.js:21]: Little typo on line 21, teaching has the g separated from the word.

**Review by Ryan-E-Mark — COMMENTED** [review]
Looking great so far! Good job with the styling to match the mockup and using the button component to keep things consistent. There is a small typo in the first paragraph with "teaching" typed out as "teachin g", which will be a quick fix.

Additionally you could add an onClick function for the "BROWSE PROGRAMS" button that redirects to "/parent-booking" if the user is logged in or to "/login" if they aren't logged in. 

Overall, Awesome job getting this landing page built out! Excited to see it once the background image is finished.

File: src/components/pages/Landing/RenderLandingPage.js [hunk 2]
```diff
@@ -1,19 +1,75 @@
...
+            </p>
+          </div>
+          <Button
+            classType="landing-second-container-btn"
+            className="landing-second-container-btn"
+            buttonText="VISIT WEBSITE"
+          ></Button>
+        </div>
       </div>
+      <p></p>
```
Comment (MatthewMion) [inline src/components/pages/Landing/RenderLandingPage.js:72]: empty p tag here

**Review by MatthewMion — COMMENTED** [review]
Look greats, only found an empty p tag at the bottom

**Review by pjaepole — COMMENTED** [review]
Thank you for the pull request.
Remove the package lock.json from the pull request,
Do make updates on the Trello board and make comments and share your progress with the rest of the team.
Great job matching the style from the mock-up,

File: src/components/pages/Landing/RenderLandingPage.js (hunk 1 above)
Comment (ElchinNasirov) [reply src/components/pages/Landing/RenderLandingPage.js]: Thank you for reviewing my code. I fixed the typo that you mentioned above. 
https://github.com/BloomTech-Labs/coder-heroes-fe/pull/103/commits/c75a86413ed87d6f2f0ee1bb2bc87d56976ebf42

File: src/components/pages/Landing/RenderLandingPage.js (hunk 2 above)
Comment (ElchinNasirov) [reply src/components/pages/Landing/RenderLandingPage.js]: Thanks fro reviewing the code. I fixed the typo. You can see it here => https://github.com/BloomTech-Labs/coder-heroes-fe/pull/103/commits/c75a86413ed87d6f2f0ee1bb2bc87d56976ebf42

File: package.json [hunk 3]
```diff
@@ -81,6 +81,7 @@
     "@testing-library/user-event": "^7.2.1",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-react-hooks": "^4.0.4",
+    "file-loader": "^6.2.0",
```
Comment (paulstgermain) [inline package.json:84]: Test comment

File: src/components/pages/Landing/RenderLandingPage.js [hunk 4]
```diff
@@ -1,18 +1,64 @@
...
-          This is an example of how we'd like for you to approach page/routable
-          components.
-        </p>
-        <p>
-          <Link to="/example-list">Example List of Items</Link>
-        </p>
+    <div className="landing-container">
+      {/* Main text container */}
+      <div className="landing-main-container">
+        <div className="landing-main-text">
```
Comment (paulstgermain) [inline src/components/pages/Landing/RenderLandingPage.js:10]: Test comment #2

**Review by paulstgermain — COMMENTED** [review]
Finished review

**Review by Ryan-E-Mark — APPROVED** [review]

**Review by RoseGabriely — COMMENTED** [review]
Please make sure to update the coderheros logo from diagonal to horizontal :)

**Review by MatthewHoseki — APPROVED** [review]

**Review by MatthewHoseki — APPROVED** [review]

**Review by MatthewHoseki — APPROVED** [review]

**Review by MatthewHoseki — APPROVED** [review]

**Review by MatthewMion — APPROVED** [review]

**Review by pjaepole — APPROVED** [review]

**Review by seanmp3 — APPROVED** [review]
Looks nice! Styling is responsive and landing page tells a lot about the product in a short amount of text. Keep up the good work man!

**Pull request merged**